packages/origin-backend/src/pods/device/device.controller.ts

```diff
@@ -46,11 +46,26 @@ export class DeviceController {
         return this.deviceService.getAll();
     }
 
+    @Get('/myDevice')
```

Comment:
We don't use camelCase for URLs, but we use dash-separated URLs. Let's use `/my-devices` instead of `/myDevice`.